lib/features/academic_calendar_greeting/model/academic_week_exception.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:collection/collection.dart';
+
+import '../../../utils/datetime_utils.dart';
+import '../repository/academic_calendar_repo.dart';
+import 'academic_day.dart';
+import 'weekday_enum.dart';
+
+extension AcadWeekExceptionExtraAttrs on List<AcademicWeekException> {
+  bool _checkIfThisIsToday(AcademicWeekException element) =>
+      element.day.isSameDay(now);
+
+  bool get isTodayAnException {
+    return any(_checkIfThisIsToday);
+  }
+
+  AcademicDay? get changedDay {
+    final changedDayData = firstWhereOrNull(_checkIfThisIsToday);
+    if (changedDayData == null) return null;
```

Comment:
I think your syntax here is nice and clean but we use here `ternary conditional operato`r  as well. Just matter of preference :-) I personally like it :) 

`return changedDayData != null ? 
`
` AcademicDay(`
      `      isEven: changedDayData.changedDayIsEven,`
   `         isExamSession: false,`
  `          isHolidays: false,`
  `          weekday: WeekdayEnum.fromJson(changedDayData.changedWeekday),`
`) : null`

Reply:
Yep, I sometimes like the classic `if` :P ; but I'll do as proposed, it will look cool here I think.